src/Library/demos/Breakpoints/main.blp

```diff
@@ -0,0 +1,57 @@
+using Gtk 4.0;
+using Adw 1;
+
+Gtk.Window {
```

Comment:
should we have a title/subtitle like others demo?

We can also add a text that says "Resize the window to see the breakpoint effect"